plugin/pkg/scheduler/metrics/metrics.go

```diff
@@ -44,6 +42,22 @@ var (
 			Buckets:   prometheus.ExponentialBuckets(1000, 2, 15),
 		},
 	)
+	SchedulingAlgorithmPredicateEvaluationDuration = prometheus.NewHistogram(
```

Comment:
I agree it would be awesome to see time taken for all predicates, but in reality since some of them are pretty fast (i.e. I assume NodeFitsCPU or Memory is probably a constant time operation?),  . deciding wether to drill down that far ; is a separate pull request.